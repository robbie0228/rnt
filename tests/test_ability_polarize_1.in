board
ability 5 a
move a down
board
ability 3 A
move A up
board
move b down
move A up
move b down
move A up
board
move b down
board
ability 5 b
quit
