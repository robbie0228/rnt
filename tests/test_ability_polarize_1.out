Player 1:
Downloaded: 0D, 0V
Abilities: 5
a: D1   b: D2   c: D3   d: D4
e: V1   f: V2   g: V3   h: V4
========
abcSSfgh
...de...
........
........
........
........
...DE...
ABCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: ?    B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

Player 1:
Downloaded: 0D, 0V
Abilities: 4
a: ?    b: ?    c: ?    d: ?
e: ?    f: ?    g: ?    h: ?
========
.bcSSfgh
a..de...
........
........
........
........
...DE...
ABCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: D1   B: D2   C: D3   D: D4
E: V1   F: V2   G: V3   H: V4

Player 1:
Downloaded: 0D, 0V
Abilities: 4
a: V1   b: D2   c: D3   d: D4
e: V1   f: V2   g: V3   h: V4
========
.bcSSfgh
a..de...
........
........
........
A.......
...DE...
.BCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 4
A: ?    B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

Player 1:
Downloaded: 0D, 0V
Abilities: 4
a: V1   b: D2   c: D3   d: D4
e: V1   f: V2   g: V3   h: V4
========
..cSSfgh
A..de...
.b......
........
........
........
...DE...
.BCSSFGH
========
Player 2:
Downloaded: 0D, 1V
Abilities: 4
A: D1   B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

Player 1:
Downloaded: 0D, 0V
Abilities: 4
a: V1   b: ?    c: ?    d: ?
e: ?    f: ?    g: ?    h: ?
========
..cSSfgh
A..de...
........
.b......
........
........
...DE...
.BCSSFGH
========
Player 2:
Downloaded: 0D, 1V
Abilities: 4
A: D1   B: D2   C: D3   D: D4
E: V1   F: V2   G: V3   H: V4

